Test/CoreSDK.Test/Shared/Extensibility/Implementation/TaskTimerTest.cs

```diff
@@ -153,7 +153,7 @@ public void CancelsPreviousActionWhenStartIsCalledMultipleTimes()
             }
 
             [TestMethod]
-            [Timeout(1000)]
+            [Timeout(1500)]
```

Comment:
I think we're chasing a problem on the build server because there is no reason this code should take over one second to execute.  Something is wrong that is slowing everything down.